### PR TITLE
feat: [React Course]: Module React Forms

### PR DESCRIPTION
**Pull request by Margaryta-Maletz**
#### 🤔 This is a ...

- [ ] 🌟 New task
- [x] 🌐 New module
- [ ] ⚙️ Update to an existing task
- [ ] 🔧 Update to an existing module
- [ ] 🔗 Update or addition of external resources or links
- [ ] 🐛 Fix in a task or related content
- [ ] 🛠 Fix in a module or related content
- [ ] ✏️ Fixed a typo or grammatical error
- [ ] 🔗 Fixed a broken link
- [ ] ❓ Other (specify: **\*\*\*\***\_\_\_\_**\*\*\*\***)

#### Description

Add Forms module for the React course

#### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [ ] 📝 I have commented my code, particularly in hard-to-understand areas.
- [ ] 🔧 I have made corresponding changes to the documentation (if applicable).
- [ ] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

File: react/modules/forms/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+   - [article: Build Dynamic Forms with **React Hook Form**](https://claritydev.net/blog/build-dynamic-forms-react-hook-form) - [20 minutes]
+   - [article: Building dynamic forms with **Formik** with React and TypeScript](https://scottdj92.ghost.io/building-dynamic-forms-with-formik-with-react-and-typescript/) - [20 minutes]
+
+10. **How to push submitted form data to the server or to state management libraries:**
+
+    - [article: Explain the process of submitting forms and handling form submissions in Redux](https://www.geeksforgeeks.org/explain-the-process-of-submitting-forms-and-handling-form-submissions-in-redux/) - [20 minutes]
+    - [article: **React Hook Form** - Integrating with services](https://react-hook-form.com/get-started#Integratingwithservices) - [5 minutes]
+    - [article: **Formik** - Form Submission](https://formik.org/docs/guides/form-submission) - [10 minutes]
+
+## Practice 💻
```
Comment (kravaring) [inline react/modules/forms/README.md:83]: This section can be removed

File: react/modules/forms/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# [React](https://github.com/rolling-scopes-school/tasks/tree/master/react) Forms 🌟
+
+## Module Overview 📚
+
+The "React Form" module explores the most popular form libraries for React are **React Hook Form** or **Formik**. Through this module, students will learn an overview of solutions to handle forms. The module covers one of the most popular solutions to create a fully functional form with input validation.
```
Comment (kravaring) [inline react/modules/forms/README.md:5]: We will be mostly using the same practical task for Forms which we've used before.
The task gives some hands-on experience working with uncontrolled components approach when dealing with forms, as well as React Hook Form.
May be it is worth providing more resources specific for those topics and leave just a few devoted to Formik (as an extra topic)?
Topics used in the practical task:
- Uncontrolled components when using forms;
- React Hook Form;
- Yup for validation;

Link to the old module:
https://github.com/rolling-scopes-school/tasks/tree/master/react/modules/_archive/module06

**Review by kravaring — CHANGES_REQUESTED** [review]
Please, consider the suggested changes :)

File: react/modules/forms/README.md (hunk 2 above)
Comment (Margaryta-Maletz) [reply react/modules/forms/README.md:5]: Added links from old module.
Formik moved to additional materials.

File: react/modules/forms/README.md (hunk 1 above)
Comment (Margaryta-Maletz) [reply react/modules/forms/README.md]: Removed

File: react/modules/forms/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+- **7 hours**
+
+## Theory 📖
+
+Students are encouraged to explore the following resources:
+
+1. **Getting Started with React Forms:**
+
+   - [video: webDev React JS #10 Формы (Forms)](https://www.youtube.com/watch?v=LLum_dcrbFo) - [15 minutes]
```
[the remark this answers is not in the document]
Comment (Margaryta-Maletz) [reply react/modules/forms/README.md]: Done

**Pull request merged**